docs/server/generate_http_server_api_documentation.py

```diff
@@ -198,6 +198,7 @@ def pretty_json(data):
         "6qHyZew94NMmUTYyHnkZsB8cxJYuRNEiEpXHe1ih9QX3",
         "AdDuyrTyjrDt935YnFu4VBCVDhHtY2Y6rcy7x2TFeiRi"
     ]
+    res_data['public_key'] = 'NC8c8rYcAhyKVpx1PCV65CBmyq4YUbLysy3Rqrg8L8mz'
```

Comment:
I grabbed this branch and rendered the docs locally. The result was:

![screenshot from 2017-01-30 15-00-22](https://cloud.githubusercontent.com/assets/1210951/22425743/e340dbca-e6fc-11e6-8323-ac37ca40cf58.png)

Yay, no more **null**.